awx_collection/plugins/modules/workflow_job_template.py

```diff
@@ -517,68 +517,63 @@
     workflow_nodes:
       - identifier: node101
         unified_job_template:
-          name: example-project
+          name: example-inventory
           inventory:
             organization:
               name: Default
           type: inventory_source
         related:
-          success_nodes: []
           failure_nodes:
             - identifier: node201
-          always_nodes: []
-          credentials: []
-      - identifier: node201
-        unified_job_template:
-          organization:
-            name: Default
-          name: job template 1
-          type: job_template
-        credentials: []
-        related:
-          success_nodes:
-            - identifier: node301
-          failure_nodes: []
-          always_nodes: []
-          credentials: []
-      - identifier: node202
+      - identifier: node102
         unified_job_template:
           organization:
             name: Default
           name: example-project
           type: project
         related:
-          success_nodes: []
-          failure_nodes: []
-          always_nodes: []
-          credentials: []
-      - identifier: node301
-        all_parents_must_converge: false
+          success_nodes:
+            - identifier: node201
+      - identifier: node201
         unified_job_template:
           organization:
             name: Default
-          name: job template 2
+          name: example-job template
           type: job_template
-        execution_environment:
-            name: My EE
         inventory:
-          name: Test inventory
+          name: Demo Inventory
           organization:
             name: Default
         related:
+          success_nodes:
+            - identifier: node401
+          failure_nodes:
+          -   identifier: node301
+          always_nodes: []
           credentials:
               - name: cyberark
                 organization:
                     name: Default
           instance_groups:
               - name: SunCavanaugh Cloud
-              - name: default
           labels:
               - name: Custom Label
-              - name: Another Custom Label
                 organization:
                     name: Default
-  register: result
+      - all_parents_must_converge: false
+        identifier: node301
+        unified_job_template:
+            description: Approval node for example
+            timeout: 900
+            type: workflow_approval
+            name: "{{ approval_node_name }}"
```

Comment:
Did you mean to leave this templated? (going by what you said in your comment)

I'm not highly adverse to this, just wanted to call it out.

Reply:
I did not going to fix.

Reply:
Fixed